### ISSUE TITLE
pngcrush/optipng

### ISSUE THREAD

**Issue by pphaneuf** [quadra-game/quadra#78]

**Comment by pphaneuf** [comment]
There are lots of png files in quadra. The tools above (both are free sw)
shrink the file size of png images, keeping the image exactly the same.
Might worth to give a try if it has a noticeable impact on the overall .res
file size, hence package size for distros, etc.

**Comment by pphaneuf** [comment]
I think we ran pngcrush already (although this is the first I hear of optipng), but
I'm not sure. I'll look into this and see if it gives us some improvement (there
might be some things where we have to be careful not to disturb the layout of the
palette, stuff like that).

Thanks!

**Comment by pphaneuf** [comment]
I think we did already run pngcrush as best we could when we converted to PNG. Some
spot checking reveals the sizes to be identical. optipng did improve the size a
little bit, but nothing too impressive (83 bytes saved on the biggest file, 158KB),
so I decided to leave it as is for now.

Of note, removing the French version in trunk did cut out 383KB from the resource
file. I'll also try to make sure that packaging keeps the rarely-changing resource
file separated from the more-often tweaked (but small) binaries, so people don't
download a ton of stuff on every update.
